packages/system/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.67.4"
```

```suggestion
- version: "1.68.0"
```

Since its an enhancement.